pkg/scheduler/profile/profile.go

```diff
@@ -98,6 +98,7 @@ func (v *cfgValidator) validate(cfg config.KubeSchedulerProfile, f framework.Fra
 	for _, plCfg := range cfg.PluginConfig {
 		if plCfg.Name == queueSort {
 			queueSortArgs = plCfg.Args
+			break
```

Comment:
why this?

Reply:
Find the matched queueSortArgs and quick return. We can have only one pluginConfig for each plugin.

Reply:
Yes, here is to validate the `queueSort` pluginConfig, add `break` to quick return.